Sources/FluentKit/Database/Database.swift

```diff
@@ -45,6 +45,10 @@ extension Database {
     public var history: QueryHistory? {
         self.context.history
     }
+
+    public var maxPerPage: Int? {
```

Comment:
The name is a little ambiguous. Something like `pageSizeLimit` would be more self-explanatory.

Reply:
@calebkleveter I can get on board with that. @pulpn0ir what about you?

Reply:
Fully agree. I updated both PRs.